jmclient/jmclient/wallet.py

```diff
@@ -2239,6 +2282,16 @@ def timestamp_to_time_number(cls, timestamp):
             raise ValueError("datetime out of range")
         return timenumber
 
+    @classmethod
+    def timestamp_to_time_number(cls, timestamp):
+        """
+        converts a unix timestamp to a time number
+        """
+        #workaround for the year 2038 problem on 32 bit systems
+        #see https://stackoverflow.com/questions/10588027/converting-timestamps-larger-than-maxint-into-datetime-objects
+        dt = datetime.utcfromtimestamp(0) + timedelta(seconds=timestamp)
+        return cls.datetime_to_time_number(dt)
```

Comment:
This rewrite is erroring for me in tests with:

```
>       dt = datetime.utcfromtimestamp(0) + timedelta(seconds=timestamp)
E       TypeError: unsupported type for timedelta seconds component: datetime.datetime
```

(Note I'm using py3.6 there, mentioning only in case the error is version specific).

Reply:
yes, fixed in 9763d11

Reply:
Confirmed fixed as of 9763d11f25d97cb442c4470eedb56ccf685318c1